src/pick.c

```diff
@@ -179,7 +174,6 @@ main(int argc, char **argv)
 	}
 
 	free(choices.v);
-	free(input.string);
```

Comment:
Since we're only ever freeing this at the end of successful execution, I agree that letting the OS do this is fine.